srai/regionizers/voronoi_regionizer.py

```diff
@@ -36,20 +38,21 @@ def __init__(self, seeds: gpd.GeoDataFrame, max_meters_between_points: int = 10_
 
         Args:
             seeds (gpd.GeoDataFrame): GeoDataFrame with seeds for
-                creating a tessellation.
+                creating a tessellation. Minimum 4 seeds are required.
+                Seeds cannot lie on a single arc.
             max_meters_between_points (int): Maximal distance in meters between two points
                 in a resulting polygon. Higher number results lower resolution of a polygon.
 
         Raises:
-            ValueError: If seeds are laying on a single arc.
             ValueError: If any seed is duplicated.
+            ValueError: If less than 4 seeds are provided.
```

Comment:
To be honest - I'm confused why I cannot supply 1, 2 or 3 points. In theory it should work. Can there be any workarounds implemented to make it possible?

Reply:
Problem arises from matrix rank - 3 points are on a single plane, 4 points will extrapolate it into multiple dimensions. We can add 2D voronoi regionizer for another use case.